### DISCUSSION TITLE
Improve the scene start logic

### DISCUSSION THREAD

**Discussion by samme** [photonstorm/phaser#5999] [Feature Requests]
-  Scenes that are `LOADING` should ignore any start operations
-  Scenes that are `SLEEPING` should be shut down before starting (just like `PAUSED` scenes)

These changes would avoid the problem where a scene's `create()` method is called twice, usually duplicating things.

Examples: https://codepen.io/samme/pen/rNYyxyQ

**Comment by photonstorm** [comment]
Agreed and merged!
